src/compiler/diagnosticMessages.json

```diff
@@ -5824,6 +5824,11 @@
         "category": "Message",
         "code": 6929
     },
+    "`false` unless target is ESNext or above ES2022": {

```

Comment:
I'd prefer either

* `true` for ES2022 and above, including ESNext.
* `false` except for ES2022 and above, including ESNext.

and I think it's quite plausible to drop 'including ESNext'.

Also, minor, but: do any other descriptions use backticks? 

I know that no (or few) error messages do, but these descriptions are their own thing.